lib_eio_windows/dune

```diff
@@ -1,12 +1,13 @@
 (library
  (name eio_windows)
  (public_name eio_windows)
- (library_flags :standard -ccopt -lbcrypt -ccopt -lntdll)
+ (library_flags :standard -cclib -lbcrypt -cclib -lntdll)
```

Comment:
Fixed to use `-cclib` as the `-l` options are for the linker rather than the compiler.